curriculum/challenges/english/blocks/workshop-breadth-first-search/68da604b8b6012a33e2344fb.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 68da604b8b6012a33e2344fb
+title: Step 1
+challengeType: 20
+dashedName: step-1
+---
+
+# --description--
+
+In this workshop, you'll implement a function that generates all valid combinations of parentheses using a breadth-first search (BFS) approach.
```

Comment:
It might be good to show an example of all the valid combinations / permutations of parentheses here as an example. We could say something like `For example, the valid combinations of two parentheses are ( ( ) ) and ()().`